electrical/electrical_build.md

```diff
@@ -188,11 +188,11 @@ The below table and figure shows an approximate amount of length of wire you wil
 
 Section | Wires Needed | Length [inches]
 ------- | ------------ | --------------- 
-A | 2x | 20AWG & 4x 30AWG | 35
-B | 2x | 20AWG & 4x 30AWG | 28
-C | 2x | 20AWG & 4x 30AWG | 48
-D | 2x | 20AWG & 3x 30AWG | 20
-E | 2x | 20AWG & 3x 30AWG | 32
+A | 2x | 20AWG & 4x 24AWG | 35
+B | 2x | 20AWG & 4x 24AWG | 28
+C | 2x | 20AWG & 4x 24AWG | 48
+D | 2x | 20AWG & 3x 24AWG | 20
+E | 2x | 20AWG & 3x 24AWG | 32
```

Comment:
So this table is actually broken on your branch now. The rows need to be like `A | 2x 20AWG & 4x 24AWG | 35` (delete the extra pipe character)

![image](https://user-images.githubusercontent.com/9410259/119278630-4da7b300-bbdb-11eb-9cce-a3d760f8cbbd.png)

Reply:
Was this ok before? I only changed the numbers.

Reply:
Looks ok now.